content/references/rippled-api/public-rippled-methods/server-info-methods/server_info.md

```diff
@@ -288,6 +494,7 @@ The `info` object may have some arrangement of the following fields:
 | `complete_ledgers`                  | String                    | Range expression indicating the sequence numbers of the ledger versions the local `rippled` has in its database. This may be a disjoint sequence, for example `24900901-24900984,24901116-24901158`. If the server does not have any complete ledgers (for example, it just started syncing with the network), this is the string `empty`. |
 | `hostid`                            | String                    | On an admin request, returns the hostname of the server running the `rippled` instance; otherwise, returns a single RFC1751 word based on the node public key. |
 | `io_latency_ms`                     | Number                    | Amount of time spent waiting for I/O operations, in milliseconds. If this number is not very, very low, then the `rippled` server is probably having serious load issues. |
+| `jq_trans_overflow`                 | String - Number           | The number of times this server has had over 250 transactions waiting to be processed at once. A large number here may mean that your server is unable to handle the transaction load of the XRP Ledger network. For detailed recommendations of future-proof server specifications, see [Capacity Planning](capacity-planning.html). [New in: rippled 0.90.0][]  |
```

Comment:
"...has had more than the hard-coded limit of transactions waiting to be processed at one (currently 250)."

Maybe?

Reply:
I think that feels like too much information. This is already a dense enough description without going into what's hard-coded or not.